onchain/.yarnrc.yml

```diff
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-3.6.4.cjs
```

Comment:
@guidanoli Do you wave the same `uarnPath: .yarn/releases/yarn-3.6.4.cjs` generated line?

Reply:
Yes.